_component_design/tables-nested.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Tables - Nested
+parent: Tables
+layout: component
+category: Components
+usage: >
+  Nested tables are a specialized sub pattern of basic tables. Nested tables come in a variety of forms and allow users to drill into additional data within a table row. By hiding this data behind a user interaction we maintain the low cognitive load goals of the table pattern, while allowing for more data rich layouts.
+preview-image: preview-images/tables-nested.svg
+resource: true
+last-modified: 2017-08-17
+---
+
+{% include toc.html %}
+
+## Introduction
+{: .hxSectionTitle}
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to use
+{: .hxSubSectionTitle}
+Nested tables allow the nesting of additional levels of data within a table.
+
+
+### Best practice
+{: .hxSubSectionTitle}
+
+- All rows should be closed by default.
+- The expanded area inherits the styling of the parent row. For example, bound or unbound, and zebra stripe color.
+- There may only be one expandable content area per row.
+- Expand may be triggered in three ways:
+	- Clicking anywhere within the row that is not a link.
+	- Clicking the down-carat icon.
+	- Clicking a badge icon.
+- Collapse may be triggered in two ways:
+	- Clicking anywhere in the parent row.
+	- Clicking directly on the collapse icon (either in a parent row or at the bottom of an expanded area).
```

Comment:
Move the triggering bullets out into their own list. IE, the nested bullets here make the section hard to read.  Consider making these H4s above their lists:  
Triggering Expand:
Triggering Collapse: